src/components/Match/CrossTable.jsx

```diff
@@ -16,6 +16,8 @@ import constants from '../constants';
 
 const teamIconStyle = { width: '30px', height: '30px' };
 
+const default_hero = { name: 'Unknown Hero' };
```

Comment:
Move the english string to langs/en-US.json (don't need to do the other languages manually)